doc/api-extensions.md

```diff
@@ -2321,3 +2321,6 @@ Calling `POST /1.0/storage-pools/<pool>/custom/<volume>?target=<target>` will mo
 
 ## `disk_io_bus`
 This introduces a new `io.bus` property to disk devices which can be used to override the bus the disk is attached to.
+
+## `cephfs_create_missing`
```

Comment:
When picking an API extension name I look to the past extensions to try and ensure my name is consistent with the past naming styles.

In this case a search for `cephfs` in the api.go file shows two past extensions:

```
	"storage_cephfs",
	"storage_cephfs_fscache",
```

So I think `storage_cephfs_create_missing` would be more in line with past naming.